### DISCUSSION TITLE
Connection Not working

### DISCUSSION THREAD

**Discussion by LukasKarer** [angristan/wireguard-install#287] [Help]
Hello guys,
 if I am trying to connect to the server that i setuped with the script all my connection handshakes are not recieving any answers. I really don't know what to do..
Someone has got any idea what could be the error?
Thank u!

**Comment by houmie** [comment]
Are you using IPv4 or IPv6?   IPv4 works.  IPv6 doesn't seem to work.

**Comment by ghost** [comment]
Did you check your firewall settings? If you are on Ubuntu, "ufw status" to see whether your VPN's port is on the list? If not, "ufw allow <PORT_NUM>" to enable it.
Other than that, some VPS providers may require to manually enable firewall from their portal.
